src/ParcelRegistry.Projections.LastChangedList/LastChangedListProjections.cs

```diff
@@ -11,7 +11,7 @@ namespace ParcelRegistry.Projections.LastChangedList
     [ConnectedProjectionDescription("Projectie die markeert voor hoeveel percelen de gecachte data nog geüpdated moeten worden.")]
     public class LastChangedListProjections : LastChangedListConnectedProjection
     {
-        private static readonly AcceptType[] SupportedAcceptTypes = { AcceptType.Json, /*AcceptType.JsonLd,*/ AcceptType.Xml };
+        private static readonly AcceptType[] SupportedAcceptTypes = { AcceptType.Json, AcceptType.Xml, AcceptType.JsonLd };
```

Comment:
Can you pls test with the other registries if local, local docs, public & public docs are OK?

Reply:
I don't understand the question.